### PR TITLE
feat: add network_id to outputs

### PR DESCRIPTION
**Pull request by MXfive**
### Description

* Add VPC Network ID to outputs

---
This change is [Reviewable](https://reviewable.io/reviews/honestbank/terraform-gcp-vpc/4)

### CODE REVIEW

File: vpc/outputs.tf [hunk 1]
```diff
@@ -3,6 +3,12 @@ output "network_name" {
   value       = google_compute_network.vpc.name
 }
 
+output "network_id" {
```
Comment (bludot) [inline vpc/outputs.tf:6]: It isn’t clear if this is gcp or aws. Maybe prefix it?

Comment (bludot) [reply vpc/outputs.tf:6]: Nvm just saw it’s in gcp module

**Review by bludot — APPROVED** [review]

File: outputs.tf [hunk 2]
```diff
@@ -3,6 +3,11 @@ output "vpc_network_name" {
   value       = module.vpc.network_name
 }
 
+output "vpc_network_id" {
+  description = "The id of the VPC."
```
Comment (jai) [inline outputs.tf]: id > ID

File: vpc/outputs.tf [hunk 3]
```diff
@@ -3,6 +3,12 @@ output "network_name" {
...
+  description = "The id of the VPC."
```
Comment (jai) [inline vpc/outputs.tf]: id > ID

File: vpc/outputs.tf [hunk 4]
```diff
@@ -3,6 +3,12 @@ output "network_name" {
...
+  value       = google_compute_network.vpc.id
+}
+
```
Comment (jai) [inline vpc/outputs.tf:10]: one too many newlines?

**Review by jai — APPROVED** [review]
LGTM

**Pull request merged**